packages/cloudflare/src/cli/build/bundle-server.ts

```diff
@@ -136,7 +138,9 @@ export async function bundleServer(buildOpts: BuildOptions): Promise<void> {
       // We make sure that environment variables that Next.js expects are properly defined
       "process.env.NEXT_RUNTIME": '"nodejs"',
       "process.env.NODE_ENV": '"production"',
-      "process.env.NEXT_MINIMAL": "true",
+      "process.env.TURBOPACK": "false",
```

Comment:
can you add comments explaining the effect of `TURBOPACK` and `_NEXT_EXPERIMENTAL_REACT`?
Thanks!